deps/undici/src/lib/observable.js

```diff
@@ -0,0 +1,802 @@
+/*
+Ported from work here:
+https://github.com/WICG/observable
+
+MIT License
```

Comment:
Note that I added this to the original reference impl just because I _personally_ didn't want to be liable for anything if someone used it and something bad happened. I'm personally okay with removing this license statement and moving it over to whatever license node is under. As long as @domfarolino agrees to that.